### DISCUSSION TITLE
Set precise size of brush.

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#541] [Ideas]
I feel that it would get to allow ablilty to size size of brush with number. I sometimes know what size I want  but can't get the number because when I slide to select it, it skips over the number I want. It gets annoying. I feel been able to type the size would be great.

**Comment by Lioncub901** [comment]
Can you add this to 2.0. I feel it would not be that hard.

**Reply by yaknepper** [reply]
Yes please!

**Comment by DinoAwakening** [comment]
I would say to you guys to set a minimum and maximum size for your brushes to make it easier to land on the size you’re looking for. Although, I totally agree with you all to set it with something more precise like a numeric setting or arrows rather than the slide bar we currently have for almost everything in ToonSquid. BUT, I also understand it has something to do with the style of the app itself, it might be hard to adapt it onto the app while looking as good as the slide bar.

**Reply by DinoAwakening** [reply]
<img width="802" alt="image" src="https://github.com/keiwando/toonsquid/assets/120270913/036cd9be-5f4a-47f5-a78c-212db0306515">

Something like this perphaps?

**Reply by DWrightAnimation** [reply]
I'd say having the additional options be available outside of the brush editor would be useful, but may I suggest adding one for smoothing?

**Reply by DinoAwakening** [reply]
Yeah, sure! Just have to figure out how to implement it correctly xD

**Reply by DWrightAnimation** [reply]
Also, was this a photoshop edit, or is this a mod of the app?

**Reply by DinoAwakening** [reply]
I’m used to make concept edits in my spare time xd

**Reply by DWrightAnimation** [reply]
![image](https://github.com/keiwando/toonsquid/assets/129681998/4d70165d-490e-4c2a-97b5-5caaa0eb8e20)

I hope you don't mind, but I took the concept you made and made a few simple edits to make it something that would probably be more likely to work in ToonSquid.

**Reply by DinoAwakening** [reply]
I thought the thing was not having to a slide bar ever again since a precise number is more convinient. Like, you could “slide” over that number form left to right and viceversa to move way further than using the arrows. Basically, I took the concept from RoughAnimator (which is by far superior in that aspect than ToonSquid).

Postdata: 4th Attempt requesting Keiwando to make a rework to the Fill Bucket Tool, and make it work like RoughAnimator’s one XD

**Comment by tbrainerd** [comment]
A related suggestion. When I adjust brush size with my Apple Pencil, I can't see the preview because it's right under my hand. Perhaps the preview could appear off to the side somewhere. I think this would be better regardless of being right- or left-handed.
